Sources/AWSSDKSwiftCore/AWSClient.swift

```diff
@@ -432,7 +474,14 @@ extension AWSClient {
                 .applyMiddlewares(serviceConfig.middlewares + self.middlewares)
                 .createHTTPRequest(signer: signer)
         }.flatMap { request in
-            return self.invoke(request, with: serviceConfig, on: eventLoop, logger: logger, stream: stream)
+            return self.invoke(
+                request,
+                with: serviceConfig,
+                on: eventLoop,
+                logger: logger,
+                context: context,
```

Comment:
Shouldn't `context` be the last parameter?

Reply:
it should be the last not optional, per https://github.com/slashmo/gsoc-swift-baggage-context#argument-namingpositioning

in AWS SDK eventLoop is sometimes optional and sometimes not which affects the order, hmm

Reply:
`stream` is the last non-optional argument